src/Simplex/Chat/Store/Postgres/Migrations/M20250217_superpeers.hs

```diff
@@ -0,0 +1,46 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Store.Postgres.Migrations.M20250217_superpeers where
+
+import Data.Text (Text)
+import qualified Data.Text as T
+import Text.RawString.QQ (r)
+
+m20250217_superpeers :: Text
+m20250217_superpeers =
+  T.pack
+    [r|
+CREATE TABLE superpeers(
+  superpeer_id BIGINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  address TEXT NOT NULL,
+  name TEXT NOT NULL,
+  domains TEXT NOT NULL,
```

Comment:
why plural?

Reply:
and probably not needed - attaching to operators happens on the fly, based on pattern match anyway.

Reply:
why can't it have plural - same as servers?

> and probably not needed - attaching to operators happens on the fly, based on pattern match anyway.

I thought we agreed above to make it a separate field

Reply:
here it's used as just a mechanism to link to operators - for server its host directly implies its domain, but for superpeer domain of server's host in address doesn't necessarily belong to same operator (simply speaking, technically operator can host link to superpeer not on their own smp server)

Reply:
so for example each superpeer can be preconfigured with all the same domains that a preconfigured operator has